examples/rcbilling-demo/src/tests/main.test.ts

```diff
@@ -43,7 +43,7 @@ async function setupTest(
   userId: string,
 ): Promise<{ browser: Browser; page: Page }> {
   const browser = await puppeteer.launch({
-    headless: "new",
+    headless: true,
```

Comment:
Looks like this was a breaking change in the latest puppeteer major so I had to change it.